rotkehlchen/tests/exchanges/test_ftx.py

```diff
@@ -0,0 +1,567 @@
+from http import HTTPStatus
+from json.decoder import JSONDecodeError
+from unittest.mock import patch
+import warnings as test_warnings
+
+import pytest
+import requests
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.assets.converters import asset_from_ftx
+from rotkehlchen.constants.assets import A_ETH, A_USD, A_USDC
+from rotkehlchen.errors import RemoteError, UnknownAsset, UnsupportedAsset
+from rotkehlchen.exchanges.ftx import FTX
+from rotkehlchen.exchanges.data_structures import AssetMovement, Trade
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.mock import MockResponse
+from rotkehlchen.typing import AssetMovementCategory, Location, TradeType
+from rotkehlchen.utils.serialization import jsonloads_dict
+
+
+TEST_END_TS = 1617382780
+
+
+def test_name():
+    exchange = FTX('a', b'a', object(), object())
+    assert exchange.name == 'FTX'
+
+
+def test_ftx_exchange_assets_are_known(mock_ftx):
+    request_url = f'{mock_ftx.base_uri}/api/markets'
+    try:
+        response = requests.get(request_url)
+    except requests.exceptions.RequestException as e:
+        raise RemoteError(
+            f'FTX get request at {request_url} connection error: {str(e)}.',
+        ) from e
```

Comment:
Hmmm why not do this request from inside the mock ftx? Isn't it just a norma api endpoint? It's just a public one that does not need authentication.